source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -77,6 +99,33 @@ content: |
              .build();
         
            ClientEncryption clientEncryption = ClientEncryptions.create(clientEncryptionSettings);
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+
+            const connectionString = "mongodb://localhost:27017";
+            const keyVaultNamespace = "encryption.__keyVault";
+            const client = new MongoClient(process.env.MONGO_URL, {
```

Comment:
Let's remove `process.env.MONGO_URL` and use `connectionString` instead

Reply:
+1, was wondering whether the `connectionString` was going to be used in the code snippet.

Reply:
Good catch, fixed!